addons/website_forum/static/src/js/website_forum.js

```diff
@@ -334,92 +517,18 @@ odoo.define('website_forum.website_forum', function (require) {
-        maximumSelectionSize: 5,
-        lastsearch: [],
-        createSearchChoice: function (term) {
-            if ($(lastsearch).filter(function () { return this.text.localeCompare(term) === 0;}).length === 0) {
-                //check Karma
-                if (parseInt($("#karma").val()) >= parseInt($("#karma_edit_retag").val())) {
-                    return {
-                        id: "_" + $.trim(term),
-                        text: $.trim(term) + ' *',
-                        isNew: true,
-                    };
-                }
-            }
-        },
-        formatResult: function(term) {
-            if (term.isNew) {
-                return '<span class="badge badge-primary">New</span> ' + _.escape(term.text);
-            }
-            else {
-                return _.escape(term.text);
-            }
-        },
-        ajax: {
-            url: '/forum/get_tags',
-            dataType: 'json',
-            data: function(term) {
-                return {
-                    q: term,
-                    l: 50
-                };
-            },
-            results: function(data) {
-                var ret = [];
-                _.each(data, function(x) {
-                    ret.push({ id: x.id, text: x.name, isNew: false });
-                });
-                lastsearch = ret;
-                return { results: ret };
-            }
-        },
-        // Take default tags from the input value
-        initSelection: function (element, callback) {
-            var data = [];
-            _.each(element.data('init-value'), function(x) {
-                data.push({ id: x.id, text: x.name, isNew: false });
-            });
-            element.val('');
-            callback(data);
-        },
-    });
-
-    $('textarea.load_editor').each(function () {
-        var $textarea = $(this);
-        var editor_karma = $textarea.data('karma') || 30;  // default value for backward compatibility
-        if (!$textarea.val().match(/\S/)) {
-            $textarea.val("<p><br/></p>");
-        }
-        var $form = $textarea.closest('form');
-        var toolbar = [
-                ['style', ['style']],
-                ['font', ['bold', 'italic', 'underline', 'clear']],
-                ['para', ['ul', 'ol', 'paragraph']],
-                ['table', ['table']],
-                ['history', ['undo', 'redo']],
-            ];
-        if (parseInt($("#karma").val()) >= editor_karma) {
-            toolbar.push(['insert', ['link', 'picture']]);
-        }
-        $textarea.summernote({
-                height: 150,
-                toolbar: toolbar,
-                styleWithSpan: false
-            });
+    },
+    /**
+     * @private
+     * @param {Object} ev
+     */
+    _onValidatePost: function (ev) {
+        var $link = $(ev.currentTarget);
+        var refreshCountPost = $('#count_post').val() - 1;
+        $('#count_post').val(refreshCountPost);
+        $link.parent().parent().parent().fadeOut();
```

Comment:
This is particularly ugly so I tried to check how it was done before... and I cannot find the equivalent code in the old code. Where did you find these 4 lines ?